### ISSUE TITLE
🚧 Dev ➾ reconcile skipped tests in e2e with bug reports

### ISSUE THREAD

**Issue by michaelkernaghan** [ecadlabs/taqueria#1754]
### 🚥 Definition of Done

- [ ] 🎯 Planning and Design
- [ ] 🏎️ Dev Implementation Complete
- [ ] ♾️ PR Merged
- [ ] 🏆 Product Acceptance

### 🧰 Work Description

 Reconcile skipped tests in e2e with bug reports.

ensure each skipped test is referenced to a bug report
ensure each skipped test is a bug that can be created manually in a pre-release

The skipped tests are:
  Archetype Plugin 
    compile will compile one contract under the contracts folder without a parameter
          - https://github.com/ecadlabs/taqueria/issues/1678
   compile will compile multiple contracts in the contracts folder
          - https://github.com/ecadlabs/taqueria/issues/1678
  compile can use different versions of the Archetype image
          - hangs waiting to execute the image command
          - this works fine manually, so its a test problem

  Taquito Plugin 
        - blocked by https://github.com/ecadlabs/taqueria/issues/1635

   instantiate-account will give contextual help
  -- works manually
   -- might not be working in tests because of the hyphen in the task name

    /* No longer possible to have missing network in config */
    deploy will error if an invalid network name is in the environment configuration

 Flextesa Plugin  
   start sandbox will initiate a baker daemon by default
   -- // config does not include baker by default - is this a bug?

  Contract Types Plugin
  //need to add a comment about 1635
   - generate types offers contextual help
   -  gen offers contextual help
these work in a manual test of pre-release v0.25.23-rc
perhaps the automated tests have a problem with the space in the task name
   
 Tezos-Client Plugin
    typecheck will check a contract in the artifacts folder
   -// blocked by https://github.com/ecadlabs/taqueria/issues/1671

    typechecker will check all contracts under artifacts folder
// blocked by https://github.com/ecadlabs/taqueria/issues/1671

    typecheck willcheck multiple (but not all) contracts using typecheck [sourceFile1] [sourceFile2] command
// blocked by https://github.com/ecadlabs/taqueria/issues/1671

    different tezos client image can be used
// hangs for a long time waiting for the image name
   this passes manually in pre-release v0.25.23-rc

### ✅ To Do

Reconcile the  skipped tests and bugs lists

### ⚖️ Acceptance Criteria

✨ Functional: 
 bug issue for each skipped test
comment on each skipped test pointing to the bug report

### 🔭 Scope

🙅‍♂️ Out of Scope: 
- ... 
 
 tests of plugin functions

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct
